src/components/collection-card/index.tsx

```diff
@@ -24,6 +25,7 @@ function CollectionCard({
 	heading,
 	description,
 	productsUsed,
+	eyebrowSlot,
```

Comment:
Now supports an optional `eyebrowSlot` prop. If that slot isn't used, we display a tutorial count with icon.